src/vdaf.rs

```diff
@@ -468,7 +480,8 @@ mod tests {
         // Aggregators decide whether the input is valid based on the verifier messages.
         let mut output = vec![Field64::zero(); input.as_slice().len()];
         for state in states {
-            let output_share = verify_finish(state, verifiers.clone()).unwrap();
+            let output_share =
+                verify_finish::<_, _, _, NUM_SHARES>(state, verifiers.clone()).unwrap();
```

Comment:
I'm curious if all of the type parameters, including `NUM_SHARES`, could be inferred if `verifiers` had type `[VerifierMessage<Boolean<Field64>>; NUM_SHARES]`.

Reply:
Yes, you could instead do:
```
let verifiers_slice: [VerifierMessage<Field64>; NUM_SHARES] = verifiers.try_into().unwrap();
for state in states {
    let output_share = verify_finish(state, verifiers.clone()).unwrap();
<...>
```
I wanted to prove that passing in a `Vec<VerifierMessage<F>>` works, though. What would be cool is if Rust had named trait bounds, so that instead of specifying `NUM_SHARES` positionally, I could do `let output_share = verify_finish::<N = NUM_SHARES>(...);`, since the `_, _, _,` is rather ugly.

Reply:
I'm not concerened about the ugliness here, I was just curious.